engineering/methodology.md

```diff
@@ -1,131 +1,196 @@
-# Engineering Methodology
-
-Date: 2017-05-09
+# Engineering methodology
+
+Date: 2018-01-15
+
+* [Intro](#intro)
+* [OKRs](#okrs)
+* [The Big Change](#the-big-change)
+* [Kanban](#kanban)
+  * [Company Kanban](#company-kanban)
+  * [Team Kanban](#team-kanban)
+    * [How to pick next issues to worked on?](#how-to-pick-next-issues-to-worked-on)
+* [Leads Meeting](#leads-meeting)
+* [Team Meeting](#team-meeting)
+* [Standup](#standup)
+* [Demo](#demo)
+* [Keeping track of things](#keeping-track-of-things)
+* [Definitions](#definitions)
+  * [What does TODO mean?](#what-does-todo-mean)
+  * [What does Done mean?](#what-does-done-mean)
+  * [What does Blocker mean?](#what-does-blocker-mean)
 
 ## Intro
 
-These are the notes from
- - 2016-09-12 Scrum/OKR
- - 2017-05-08 Sprint #13 Retrospective
+These are the notes from:
+
+* 2017-01-09 Scrum of Scrums
+* 2017-01-10/11 Workflow discussions
+
+Meetings for the Engineering team leads at source{d}.
+It’s the replacement for the [previous workflow](https://github.com/src-d/guide/blob/abe7932e3bc58898e49a5c1c0e3e2cac6d387144/engineering/methodology.md) \w SCRUM
+
+## OKRs
+
+Company publishes OKRs every quarter.
+
+2018Q1 OKRs in https://github.com/src-d/guide/blob/general/OKRs/README.md
+
+Each team is expected to publish it’s own ORK, aligned with the company ones.
+Engineering roles are not expected to have individual objectives.
+
+## The Big Change
+
+In 2018Q1 SCRUM is replaced by a simpler, company and team-level Kanban boards,
+so that all iterations including planning/work/retrospective do not happen
+synchronously at the same weeks for all the teams.
+
+Meetings for high-level sync including product direction, vision, etc are
+proactively scheduled by each Team Lead as needed. All Leads have periodical
+hands-on meetings \w CTO.
+
 
-meetings for the Engineering team at source{d}.
+## Kanban
 
-## OKR
+Kanban board, a tool used to visualize work and optimize the flow of the work
+among the team. (source)
```

Comment:
no link in source

Reply:
Should be https://www.atlassian.com/agile/kanban#kanban-boards